src/main/java/mate/academy/spring/boot/dto/cartItem/CartItemRequestDto.java

```diff
@@ -0,0 +1,14 @@
+package mate.academy.spring.boot.dto.cartItem;
+
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
+import lombok.Data;
+
+@Data
+public class CartItemRequestDto {
+    @NotNull
+    @Positive
+    private Long bookId;
```

Comment:
```suggestion
    private long bookId;
```
let's just use primitive, we won't need NotNull